src/Farmer/Arm/Aks.fs

```diff
@@ -0,0 +1,74 @@
+[<AutoOpen>]
+module Farmer.Arm.Aks
```

Comment:
The namespace should be (I think!) `Farmer.ContainerService` (to mirror the namespace of the resource.